ray-operator/test/sampleyaml/rayservice_test.go

```diff
@@ -72,6 +72,13 @@ func TestRayService(t *testing.T) {
 				g.Expect(err).NotTo(HaveOccurred())
 				return rs.Status.NumServeEndpoints
 			}, TestTimeoutShort).Should(BeNumerically(">", 0))
+
+			// Check if all applications are running
+			g.Eventually(func(g Gomega) bool {
+				rayService, err := GetRayService(test, namespace.Name, rayServiceFromYaml.Name)
+				g.Expect(err).NotTo(HaveOccurred())
+				return AllAppsRunning(test, rayService)
+			}, TestTimeoutMedium).Should(BeTrue())
```

Comment:
Could you rewrite it to `WithTransform` form?

Reply:
Do you mean like this?
```
g.Eventually(func() (*rayv1.RayService, error) {
				return GetRayService(test, namespace.Name, rayServiceFromYaml.Name)
			}, TestTimeoutShort).Should(WithTransform(func(rayService *rayv1.RayService) bool {
				return AllAppsRunning(test, rayService)
			}, BeTrue()))
```

Reply:
Or change AllAppsRunning to take only one parameter and remove the LOG-related parts, so it can be modified to:
```
g.Eventually(RayService(test, namespace.Name, rayServiceFromYaml.Name), TestTimeoutMedium).Should(WithTransform(AllAppsRunning, BeTrue()))
```

Reply:
The second one.